src/qt/rpcconsole.cpp

```diff
@@ -861,17 +861,18 @@ void RPCConsole::clear(bool keep_prompt)
            "Use %3 and %4 to increase or decrease the font size.\n"
            "Type %5 for an overview of available commands.\n"
            "For more information on using this console, type %6.\n"
-           "\n"
-           "%7WARNING: Scammers have been active, telling users to type"
-           " commands here, stealing their wallet contents. Do not use this console"
-           " without fully understanding the ramifications of a command.%8")
+           "\n")
             .arg(PACKAGE_NAME,
                  "<b>" + ui->clearButton->shortcut().toString(QKeySequence::NativeText) + "</b>",
                  "<b>" + ui->fontBiggerButton->shortcut().toString(QKeySequence::NativeText) + "</b>",
                  "<b>" + ui->fontSmallerButton->shortcut().toString(QKeySequence::NativeText) + "</b>",
                  "<b>help</b>",
-                 "<b>help-console</b>",
-                 "<span class=\"secwarning\">",
+                 "<b>help-console</b>") +
+        tr(
+           "%7WARNING: Scammers have been active, telling users to type"
+           " commands here, stealing their wallet contents. Do not use this console"
+           " without fully understanding the ramifications of a command.%8")
+            .arg(gArgs.GetChainName() == CBaseChainParams::TESTNET ? "<span>" : "<span class=\"secwarning\">",
```

Comment:
Wonder if it would make sense to the secwarning style to `NetworkStyle` instead of hardcoding a comparison to testnet here. The idea of that class is to make network-specific style data-driven.

Reply:
I'm not convinced that making the `secwarning` style dependent on which network is much better than replacing the styled span for a vanilla span (or even deleting the span and nested text, completely) depending on which network (and then maybe removing the `secwarning` style in those cases because it's not really used by any other string).

However, the hardcoded comparison was horrible and embarrassing so it had to go away right away, deferring doubts on the "what exactly" to some later moment after having fixed the "how".